pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -356,31 +356,41 @@ cdef class {{name}}HashTable(HashTable):
 
     @cython.boundscheck(False)
     @cython.wraparound(False)
-    def _factorize(self, const {{dtype}}_t[:] values, {{name}}Vector uniques,
-                   Py_ssize_t count_prior=0, Py_ssize_t na_sentinel=-1,
-                   object na_value=None):
+    def _unique(self, const {{dtype}}_t[:] values, {{name}}Vector uniques,
+                bint ignore_na=False, bint return_inverse=False,
+                Py_ssize_t count_prior=0, Py_ssize_t na_sentinel=-1,
+                object na_value=None):
         """
-        Calculate unique values and labels (no sorting); ignores all NA-values
+        Calculate unique values and labels (no sorting!)
 
         Parameters
         ----------
         values : ndarray[{{dtype}}]
             Array of values of which unique will be calculated
         uniques : {{name}}Vector
             Vector into which uniques will be written
+        ignore_na : boolean, default False
```

Comment:
so pls don't move the paramaters around , it is really hard to tell what changed. the new ones can all go at the end.